concent_api/core/tests/test_core_views.py

```diff
@@ -240,6 +240,49 @@ def test_send_should_return_http_400_if_task_id_already_use(self):
             error_code=ErrorCode.SUBTASK_DUPLICATE_REQUEST
         )
 
+    @freeze_time("2017-11-17 10:00:00")
+    def test_send_should_return_http_202_if_task_id_already_in_use_but_subtask_id_differs_for_the_same_requestor(self):
+
+        response_202 = self.send_request(
+            url='core:send',
+            data=dump(
+                self.correct_golem_data,
```

Comment:
what's `correct_golem_data`?

Reply:
It is defined in `setUp` of this function and used in most tests in this class.

Reply:
then I would suggest renaming it, because it doesn't say anything about what kind of message is being sent/what USE CASE is being started here...

Reply:
Because it doesn't matter which use case it is. Those tests are not supposed to test any particular use case (even if there are leftovers from testing `force report`), but some general rules among all use cases.